lib/gollum/views/helpers.rb

```diff
@@ -2,6 +2,15 @@
 
 module Precious
   module Views
+    
+    module AppHelpers
+      def extract_page_dir(path)
+        return path if @page_dir.to_s.empty?
+        @extract_path ||= "#{Pathname.new(@page_dir).cleanpath.to_s}/"
```

Comment:
The `to_s` call here is superfluous since you are using string interpolation. So `"#{Pathname.new(@page_dir).cleanpath}/"` will already end up being a String.